src/main/java/com/synopsys/integration/alert/provider/blackduck/descriptor/BlackDuckProviderDescriptorActionApi.java

```diff
@@ -74,9 +71,9 @@ public void validateConfig(final Config config, final Map<String, String> fieldE
         }
         if (StringUtils.isNotBlank(blackDuckConfig.getBlackDuckApiKey())) {
             if (blackDuckConfig.getBlackDuckApiKey().length() < 64) {
-                fieldErrors.put("blackDuckApiKey", "Not enough characters to be a Hub API Key.");
+                fieldErrors.put("blackDuckApiKey", "Not enough characters to be a Black Duck API Key.");
```

Comment:
We should just state that the API token is invalid for both cases.  We might be giving information that's too specific.